spiceaidocs/docs/secret-stores/aws-secrets-manager/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'AWS Secrets Manager Secret Store'
+sidebar_label: 'AWS Secrets Manager Secret Store'
+sidebar_position: 3
+description: 'AWS Secrets Manager Secret Store Documentation'
+---
+
+The `aws_secrets_manager` store enables Spice to read secrets from [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).
+
+```yaml
+secrets:
+  store: aws_secrets_manager
+```
+The store reads secrets named as `spice_secret_<secret-name>`, for example `dremio` login and password must be defined as `spice_secret_dremio` secret in [AWS Secrets Manager](https://console.aws.amazon.com/secretsmanager/listsecrets)
+
+<img src="/img/secrets-aws-secrets-manager-dremio-1.png" alt="" width="800" />
+
+<img src="/img/secrets-aws-secrets-manager-dremio-2.png" alt="" width="800" />
+
```

Comment:
Add a complete example of a dataset that uses secrets manager.

Reply:
Added - thank you @lukekim 

<img width="714" alt="image" src="https://github.com/spiceai/docs/assets/981580/0a211d25-dd65-4571-9059-73b8580ddc62">

Also expanded first definition to provide more clarity on where `aws_secrets_manager` should be specified.
<img width="649" alt="image" src="https://github.com/spiceai/docs/assets/981580/c3275b9f-c778-48cd-95e8-4a2ebd893d9e">